docs/wave.rst

```diff
@@ -42,7 +43,7 @@ receiver coordinates are stored in an instance of :obj:`IShot()` called :obj:`da
   data.set_receiver_pos(receiver_coords)
   data.set_shape(nt, 101)
 
```

Comment:
I mean why the spacing is defined as (20, 20). I thought in 3d it should be like (20, 20, 20) so is the origin, it define as (0, 0).